src/pages/Home.vue

```diff
@@ -154,6 +151,16 @@ export default {
     getUserIpAddress() {
       return this.sessionData ? this.sessionData.userIp : "";
     },
+
+    /** Apart from ID, are any extra inputs being validated. */
+    isExtraInputValidationsRequired() {
+      return this.getAuthType.split(",").length > 1;
+    },
+
+    /** Returns the auth methods used by each group */
+    getAuthType() {
+      return this.groupData && this.groupData.authType ? this.groupData.authType : "ID";
```

Comment:
The comment you mentioned above; combined it all into one `getAuthType`. That's what is happening. In case, the group does not have any information regarding what auth type to use, it defaults to ID.

Reply:
I'm not sure I get the `if` condition, though. If `groupData` and `groupData.authType` exist, it returns `ID`, because that's what it looks like is happening

Reply:
No, if `groupData` and `groupData.authType` exists, then `groupData.authType` is returned, otherwise `ID` is defaulted.